### PR TITLE
[release-1.32] Update contract downgrade script to be standalone

### PR DESCRIPTION
**Pull request by creydr**
Update the contract downgrade script to:
* include the main contract names already (so this can be run without parameters later from support without knowing the configmap names)
* allow to set the namespace name (so this can be run on namespaced-broker configs too)

### CODE REVIEW

**Comment by creydr** [comment]
CI issues...
/retest

**Comment by creydr** [comment]
`Could not resolve tag openshift_knative-eventing-src_knative-v1.11 in imagestream pipeline.  error=timed out waiting for the condition`

/retest

**Comment by creydr** [comment]
`Could not resolve tag openshift_release_rhel-8-release-golang-1.21-openshift-4.16 in imagestream pipeline.  error=timed out waiting for the condition`
again?

/retest

File: test/upgrade/installation/downgrade.sh [hunk 1]
```diff
@@ -1,9 +1,33 @@
...
+  cmdata=$(kubectl get cm "$cm_name" -n "$target_namespace" -ojson || true)
   if [ -n "$cmdata" ]; then
-    new_data=$(echo "$cmdata" | jq -r .binaryData.data | base64 --decode | jq 'del(.trustBundles)' -c | base64 -w 0)
+    new_data=$(echo "$cmdata" | jq -r .binaryData.data | base64 --decode | jq 'del(.trustBundles, .resources[].ingress.enableAutoCreateEventTypes)' -c | base64 -w 0)
     echo "$cmdata" | jq --arg new_data "$new_data" '.binaryData.data = $new_data' | kubectl apply -f -
   fi
 done
+
+# Trigger a reconcile of the related pods (see https://kubernetes.io/docs/tasks/configure-pod-container/configure-pod-configmap/#mounted-configmaps-are-updated-automatically)
+declare -a pods_app_labels=("kafka-broker-dispatcher" "kafka-broker-receiver" "kafka-channel-dispatcher" "kafka-channel-receiver" "kafka-source-dispatcher" "kafka-source-receiver")
```
Comment (mgencur) [inline test/upgrade/installation/downgrade.sh:30]: Should `kafka-source-receiver` actually be `kafka-sink-receiver` ? 
Testing these changes, I don't see the annotation on kafka-sink-receiver and the single test fails then:
```
--- PASS: TestServerlessUpgradeContinual/Run/KafkaBrokerContinualTest
--- FAIL: TestServerlessUpgradeContinual/Run/KafkaSinkSourceContinualTest
```

Comment (pierDipi) [inline test/upgrade/installation/downgrade.sh:30]:
```suggestion
declare -a pods_app_labels=("kafka-broker-dispatcher" "kafka-broker-receiver" "kafka-channel-dispatcher" "kafka-channel-receiver" "kafka-source-dispatcher" "kafka-sink-receiver")
```

Comment (pierDipi) [reply test/upgrade/installation/downgrade.sh:30]: Let's try

**Review by pierDipi — APPROVED** [review]
/lgtm
/approve

**Pull request merged**